src/components/_navbar.scss

```diff
@@ -107,6 +107,13 @@ nav {
     }
   }
 
+  .collapsible {
+    @include resp(small) {
+      width: 100%;
+      margin-top: 50px;
```

Comment:
For the tests to pass, you need to move-up the `margin-top` property before the `width: 100%`.

Reply:
OMG..  give me a sec. 😈

Reply:
You can see what is wrong with the tests by clicking on the red-cross and then clicking on *Travis Ci details*.

You will see in the console output a detailed error thrown by Stylelint:

![image](https://user-images.githubusercontent.com/9285210/51914086-4fc92080-23d8-11e9-860a-a05cf9477ae0.png)